elementary/clients/gcs/client.py

```diff
@@ -71,12 +73,15 @@ def get_bucket_website_url(
     ) -> Optional[str]:
         bucket_website_url = None
         if self.config.update_bucket_website:
-            full_bucket_path = (
-                f"{destination_bucket}/{bucket_name}"
-                if destination_bucket
-                else bucket_name
-            )
-            bucket_website_url = f"https://storage.googleapis.com/{full_bucket_path}"
+            if self.config.bucket_website_url:
+                bucket_website_url = self.config.bucket_website_url
+            else:
+                full_bucket_path = (
+                    f"{destination_bucket}/{bucket_name}"
+                    if destination_bucket
+                    else bucket_name
+                )
+                bucket_website_url = f"{DEFAULT_BUCKET_WEBSITE_URL}/{full_bucket_path}"
```

Comment:
I think we should use [`urljoin`](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urljoin).